mixins/form.jsx

```diff
@@ -411,7 +417,13 @@ module.exports = {
         submitting: true
       });
       submitProps = form.buildProps(props);
-      this.submit("/api/signup", submitProps, this.signupSuccess, this.signupError);
+      this.submit(url, submitProps, success, error);
     }
+  },
+  signup: function(validate, props) {
+    this.doSignup("/api/signup", validate, props, this.signupSuccess, this.signupError);
```

Comment:
I changed it, but a bit different than your initial request or my initial counter proposal.

I see you issues with the path `/api/mailchimp` it's not specific and I have no issues with making it specific. So it's not `/api/signup/mailchimp`

I did not add a `/api/signup/exacttarget` as I still like having a generic signup path that we use for 90% of our signups.

Thoughts?